node/sc/vt_recovery_test.go

```diff
@@ -186,43 +186,52 @@ func TestKLAYTransferLongRangeRecovery(t *testing.T) {
 			}
 		}
 	})
+	// TODO-Klaytn need to remove sleep
+	time.Sleep(1 * time.Second)
```

Comment:
`prepare` has a WaitGroup to sync request txs and handle txs. Do we need the `Sleep` here?

Reply:
Yes. After waiting for handle transaction, Commit() will be executed. Then updateRecoveryHint will update the status for checking.